pkg/policy/api/rule_validation.go

```diff
@@ -226,13 +225,13 @@ func (cidr CIDR) Validate() error {
 	return nil
 }
 
-// Validate validates a CIDRRule by checking that the CIDR prefix itself is
+// Sanitize validates a CIDRRule by checking that the CIDR prefix itself is
 // valid, and ensuring that all of the exception CIDR prefixes are contained
 // within the allowed CIDR prefix.
-func (c CIDRRule) Validate() error {
+func (c CIDRRule) Sanitize() error {
```

Comment:
This one is actually Validating, right?

Reply:
marked private. It is validating only for now but we don't know later on. I've only kept the `Rule` `Sanitize()` public and have hidden everything so we can rename/change this as we want in the future.